### ISSUE TITLE
Error when applying fixes with `apply-if-condition` rule

### ISSUE THREAD

**Issue by epreston** [coderaiser/putout#125]
The `apply-if-condition` rule correctly detects `Avoid empty statement in if condition` but when "--fix" is run, it returns the following error from the parser.

```
5:2  error   You passed `path.replaceWith()` a falsy node, use `path.remove  parser
```

The following code can be used to replicate the issue:

```js
const a = 1;
const b = 2;

function test() {
  if (a !== b);
}
```

**Comment by coderaiser** [comment]
Nice! Thank you :). Landed in `@putout/plugin-apply-if-condigion@1.1.0` 🎉 . Please reinstall 🐊**Putout** to get updates. Is it works for you?

**Comment by epreston** [comment]
Confirmed fixed.  Triggers `convert-comparison-to-boolean` because of the two constant expressions, evaluates and optimises away the empty if statement based on the constant expression.
